Flashcards.AnaClos/Flashcards.AnaClos/Controllers/StackController.cs

```diff
@@ -0,0 +1,136 @@
+﻿using Flashcards.AnaClos.Models;
+using Spectre.Console;
+
+namespace Flashcards.AnaClos.Controllers;
+
+public class StackController
+{
+    private ConsoleController _consoleController;
+    private DataBaseController _dataBaseController;
+
+    public StackController(ConsoleController consoleController, DataBaseController dataBaseController)
+    {
+        _consoleController = consoleController;
+        _dataBaseController = dataBaseController;
+    }
+
+    public Stack AddStack()
+    {
+        string message = "Please enter a stack name or enter # to return to main menu:";
+        string response = _consoleController.GetString(message);
+
+        if (response.Trim() == "#") return null;
+
+        var stack = new Stack{Name=response};
+        var sqlInsert = "INSERT INTO Stacks (Name) VALUES (@Name)"; 
```

Comment:
Well done parameterising your sql query here. Why didn't you do it everywhere else also? 😄